### ISSUE TITLE
Change sorting of new additions to the list

### ISSUE THREAD

**Issue by FerruccioSisti** [Pitt-CSC/Summer2021-Internships#10]
In my opinion, sorting by recent addition is a better way to approach adding new postings to this list. I would argue that majority of people who are interested in applying to internships this early are ready to start now and will be referring to this repo every so often to make sure they are as up to date as they can be with applications. It would make it easier to just scroll to the bottom of the list to see new postings that are added rather than scanning through the list or looking at new commits to find them.

**Comment by Zmwang622** [comment]
I agree. Do you want to sort it?

**Comment by Zmwang622** [comment]
@FerruccioSisti Hey! We really appreciate the feedback but wanted to see what everyone else thought. [Fill out this poll on whether we should make the postings alphabetical or sorted by dates!](https://pitt.co1.qualtrics.com/jfe/form/SV_0VD00iKfbqspNKl)

**Comment by MattChen2000** [comment]
Agreed. Personally, It would also make more sense to me to add a column recording when this job is posted/added to our list

**Comment by Zmwang622** [comment]
Yeah sounds good with me. My only concern with adding a date column would be the extra overhead of enforcing a consistent date format.

**Comment by Zmwang622** [comment]
If someone wants to make a PR with the changes, that would be sooo clutch. I'm busy this week and can't get to it this weekend.

**Comment by rweas** [comment]
In the PR I just put all the old posting dates as "Before July 8th". I think that is probably sufficient for people who are just wanting to know if there are new postings they haven't applied to.

**Comment by Zmwang622** [comment]
left a comment

**Comment by varughese** [comment]
To maintain simplicity of the initial markdown table, but also have the ability to sort new additions and such, what do you think of having a Google Sheet? Every new addition to the README would sync with the google sheet, and the google sheet could contain additional columns and have the ability to sort by newly updated

**Comment by Zmwang622** [comment]
Will probably mean having to add dates/other info on our end, but sgtm

**Comment by varughese** [comment]
We reordered the list to be by. date. This Google Sheet is synced with the repo, and every update to README.md will auto sync it: https://docs.google.com/spreadsheets/d/1bJq7YQV19TWyzPCBeQi5P4uOm8uiAAm2AHCnVNGRIDg/edit#gid=0

We also made a application tracker that also remains in sync with evertyhing, so that its easy for you to keep track of everything in one spot: https://docs.google.com/spreadsheets/d/126GcV-xVw7pZ3FIZnP2M_plDbcLwfS7HTs9ZqJHaReg/edit#gid=0

Good luck with your internship search!
